lib/ansible/modules/cloud/amazon/ec2_vpc_subnet.py

```diff
@@ -228,38 +421,49 @@ def main():
     argument_spec = ec2_argument_spec()
     argument_spec.update(
         dict(
-            az=dict(default=None, required=False),
+            az=dict(default='', required=False),
             cidr=dict(default=None, required=True),
+            ipv6cidr=dict(default='', required=False),
```

Comment:
Prefer ipv6_cidr

Reply:
This will by default delete IPv6 CIDRs, according to your example documentation

Reply:
What is the desired behavior for modules like this since users can assign and unassign IPv6 CIDR blocks to subnets at will. I figured this behavior made the most sense. This way if you have your subnets defined as a list in your vars file and you want to change or remove the CIDR you just make that change in the subnets list.

The other way would be to have a `purge_ipv6_cidr` parameter or to differentiate between `ipv6cidr is None and ipv6cidr is ''`?